strawberry/experimental/pydantic/object_type.py

```diff
@@ -73,8 +85,34 @@ def get_type_for_field(field: ModelField):
     return type_
 
 
+if TYPE_CHECKING:
+    A = TypeVar("A", bound=BaseModel)
+
+    class StrawberryTypeFromPydantic(Generic[A]):
+        """This class does not exist in runtime.
+        Its only makes the below methods visible for IDEs"""
+
+        def __init__(self, **kwargs):
+            ...
+
+        @staticmethod
+        def from_pydantic(instance: A, extra: Dict[str, Any] = None) -> A:
+            ...
```

Comment:
this is so that IDE's (well, at least my pycharm. can someone with VSCode try it out?) can discover that from_pydantic and to_pydantic exist.

Reply:
pycharm's inbuilt typechecker doesn't inference the generic parameter properly though :( . anyways pycharm users can just use the mypy plugin for that

Reply:
I'll test this on vscode!

Reply:
<img width="483" alt="image" src="https://user-images.githubusercontent.com/667029/150642684-f8da6ff7-5e42-4670-8cb9-d6f8977b162b.png">

seems to be ok!

pyright is able to find `to_pydantic` on the strawberry type, but that's because it doesn't use the mypy plugin :)